pennylane/beta/queuing/measure.py

```diff
@@ -182,11 +311,4 @@ def circuit():
         wires (Sequence[int] or int): the wire the operation acts on
     """
     # pylint: disable=protected-access
-    op = Identity(wires=wires, do_queue=False)
-
-    meas_op = MeasurementProcess(Probability)
-    qml.QueuingContext.append(op)
-    qml.QueuingContext.update_info(op, owner=meas_op)
-    qml.QueuingContext.append(meas_op, owns=op)
-
-    return op
+    return MeasurementProcess(Probability, wires=qml.wires.Wires(wires))
```

Comment:
Just as a thought...we spoke about dummy measurements before. This could be great for debugging, or when one wants to access the state of the device. Could this be done with an empty  ``MeasurementProcess``?

Reply:
Yes, I was thinking about that as well. The only issue is that currently devices won't know what to do with a measurement process that isn't a expval/prob/var/sample! So something to consider in the device refactor